tests/regression/02-base/87-casts-dep-on-param.c

```diff
@@ -1,4 +1,4 @@
-// PARAM: --set ana.activated "['base', 'mallocWrapper']" --set ana.ctx_insens[+] 'base'  --set ana.base.privatization none
```

Comment:
This may not test what we looking for when we have other analyses context-sensitive, because they may keep these contextes apart again. I'd vote for keeping this explicit list, unless we have a way to make all analyses context-insensitive.

Reply:
Sure, I can make it explicit again. As-is this test had no indication that it is strictly about no context sensitivity at all. By the way, mallocWrapper has context too, so for completeness that should be made insensitive as well.